build.cmd

```diff
@@ -1,5 +0,0 @@
-@echo off
```

Comment:
Can you describe this change a bit?

Reply:
build.cmd was removed since it is not used in any environment.

Reply:
Removing this file should be done as part of a different ticket.

See: https://jira.mongodb.org/browse/CSHARP-3246

Reply:
Brought back in favour of the other ticket.